README.md

```diff
@@ -32,6 +35,77 @@ Please read our [contributing guide](https://napari.org/dev/developers/contribut
 These steps should set you up to build and preview your docs contributions on your local machine.
 For more detailed instructions and tips, please visit the relevant sections of our [contribution guide](https://napari.org/dev/developers/contributing/documentation/index.html).
 
+#### Option 2: Using pixi (for slimfast builds only)
+
+
+[Pixi](https://pixi.sh) provides a simple, cross-platform way to quickly build the documentation using the `slimfast` target without manually installing dependencies or cloning the napari source repository.
+
+**Note: This option only supports `make slimfast` builds. For full builds with the gallery (`make html`) or other build options, use Option 1.**
+
+1. **Install pixi** (if not already installed):
+   ```bash
+   # Windows (PowerShell)
+   iwr -useb https://pixi.sh/install.ps1 | iex
+   
+   # macOS/Linux
+   curl -fsSL https://pixi.sh/install.sh | bash
+   ```
+   
+   **Important:** After installation, you need to add pixi to your PATH:
+   
+   - **Windows**: Restart your PowerShell terminal, or run:
+     ```powershell
+     $env:Path = [System.Environment]::GetEnvironmentVariable("Path","Machine") + ";" + [System.Environment]::GetEnvironmentVariable("Path","User")
+     ```
+   
+   - **macOS/Linux**: Restart your terminal, or run:
+     ```bash
+     # For bash
+     source ~/.bashrc
+     
+     # For zsh (macOS default since Catalina)
+     source ~/.zshrc
+     
+     # For fish shell
+     fish_add_path ~/.pixi/bin
+     ```
+     
+     **Note:** If you're using fish shell and pixi was just installed, you may need to manually add it to your PATH with the command above.
+   
+   Verify pixi is available by running: `pixi --version`
+
+2. **Clone this repository**:
+   ```bash
+   git clone https://github.com/napari/docs.git napari-docs
+   cd napari-docs
+   ```
+
+3. **Configure platform (REQUIRED):** In `pixi.toml`, set only ONE platform for your OS:
+   ```toml
+   platforms = ["win-64"]      # Windows
+   platforms = ["linux-64"]    # Linux
+   platforms = ["osx-64"]      # macOS Intel
+   platforms = ["osx-arm64"]   # macOS Apple Silicon
+   ```
+   Only one platform should be present in the array at a time.
```

Comment:
Why is this the case? 
My understanding was that pixi supported multiplatform solving, so that projects were truly portable, see also https://pixi.sh/dev/workspace/multi_platform_configuration/
Is it something about our docs build process that breaks this? or am I missing something?